app/Locale/German.php

```diff
@@ -174,6 +174,7 @@ public function get_locale(): array
 			'ALBUM_IMAGES' => 'Bilder',
 			'ALBUM_VIDEOS' => 'Videos',
 			'ALBUM_SUBALBUMS' => 'Unteralben',
+			'ALBUM_OWNER' => 'Owner',
```

Comment:
Just the translation:
```suggestion
			'ALBUM_OWNER' => 'Besitzer',
```